semgrep/tests/e2e/snapshots/test_dependency_aware_rules/test_dependency_aware_rules/rulesdependency_awareansi-html.yaml-dependency_awareansi.js/results.json

```diff
@@ -35,7 +35,7 @@
             "lockfile": "yarn.lock"
           }
         ],
-        "fingerprint": "34dbc50eb98e00ba192ad6f3e9716450",
+        "fingerprint": "e87e6dc13255fff5bd134556c84b5a22",
```

Comment:
@mmcqd did we expect the fingerprints to change?

Reply:
I assumed it had to do with changes you guys made between the time this branch was created and now (since I merged develop a couple times). I'm not sure specifically why they changed/if it's expected.

Reply:
well if fingerprinting changed on develop and you merged that into your branch, you'd have also merged the updated fingerprints into your branch 🤔

Reply:
Actually, I was wrong, this is expected. The target mentioned in the finding changed for all of the findings where the fingerprint changed. Because now dependency only matches are reported as being from the offending lockfile, not from the toplevel.